harness/src/main/java/org/apache/geode/perftest/yardstick/YardstickTask.java

```diff
@@ -107,6 +112,35 @@ public String defaultDescription() {
 
     runner.runBenchmark();
 
-    testDoneProbe.await();
+    Histogram lastHistogram = hdrHistogramProbe.getHistogram();
+    while (!testDoneProbe.await(10, TimeUnit.SECONDS)) {
```

Comment:
I worry abut the performance impact of calculating the histogram every 10s for logging purposes. This will effectively steal a CPU every 10s to crunch some numbers without the framework accounting for it. if work and measurement was paused for each cycle that would be ok.

Reply:
I'm not sure if there is a way we can pause yardstick during this measurement without just throwing away yardstick. I can do a test to see if there is any impact from this. I was originally thinking that since we only really care about relative performance, it doesn't matter if we impact the perf of both the baseline and the test versions.

Reply:
is calculating the histogram reliable in how long it takes? Does it take longer as we get more data? I could see how we'd account for it if it always takes the same amount of time, but a variable time doesn't seem possible to account for. I don't think we can assume that the relative performance impact would be the same between runs

Reply:
I won't fight you in throwing out yardstick. We have measured so much variability in the framework itself. We have a no-op benchmark that swings by as much last 15%. That is all noise from yardstick.